exchanges/bitstamp/bitstamp.go

```diff
@@ -53,6 +59,8 @@ const (
 // Bitstamp is the overarching type across the bitstamp package
 type Bitstamp struct {
 	exchange.Base
+
+	wsAuthToken websocketAuthResponse
```

Comment:
Can RM this doesn't look like its being used?

Reply:
1c1cc7d5dc7c1c6f34b6068d59e670c9b3eff0d9